.gitignore

```diff
@@ -2,4 +2,5 @@
 *.pyc
 .idea
 .env*
+celerybeat-schedule 
```

Comment:
Sorry I just realized that the next line has a ".db" suffix that clearly wouldn't match the bin file